src/components/UNSTABLE_Pagination/PageSelector.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file Page selector.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Select, SelectItem } from '../Select';
+import { appendComponentNamespace } from '../../globals/namespace/index';
+import { namespace as paginationNamespace } from './Pagination';
+
+const namespace = appendComponentNamespace(
+  paginationNamespace,
+  'page-selector'
+);
+
+function PageSelector({ className, id, labelText, totalPages, ...rest }) {
+  const renderPages = total => {
+    const pages = [];
+    for (let counter = 1; counter <= total; counter += 1) {
+      pages.push(
+        <SelectItem key={counter} value={counter} text={String(counter)} />
+      );
+    }
+    return pages;
+  };
+
+  return (
+    <Select
+      className={classnames(namespace, className)}
+      hideLabel
+      id={`${namespace}__input-${id}`}
+      inline
+      labelText={labelText}
+      {...rest}
```

Comment:
`...rest` or `..other`, where did we land on this? 😆

Reply:
We use both 😅 `...other` is more common in our codebase but Carbon uses `...rest almost exclusively.

Reply:
Oh! I thought we picked up the convention from them originally 😆

Reply:
Well, scratch what I said about them using `...rest` exclusively... I searched their codebase and found it was nearly half and half 😆

Reply:
Got it! I'm fine with either but we need to pick one and stick to it at this stage 😆I'm good with `other` as this is used in React's documentation - https://reactjs.org/docs/jsx-in-depth.html#spread-attributes